lib/vsc/utils/fancylogger.py

```diff
@@ -431,7 +440,7 @@ def getRootLoggerName():
         return "not available in optimized mode"
 
 
-def logToScreen(enable=True, handler=None, name=None, stdout=False):
+def logToScreen(enable=True, handler=None, name=None, stdout=False, colorize='never'):
```

Comment:
> > -def logToScreen(enable=True, handler=None, name=None, stdout=False):
> > +def logToScreen(enable=True, handler=None, name=None, stdout=False, colorize='never'):
>
> use constants for these magic strings please, even beter then None

Then it would make sense to actually use a proper enum.

But it's another additional dependency package, are you sure you want
that?